### PR TITLE
Standardize Bloom attention mask

### PR DESCRIPTION
**Pull request by wtmlon**
### Description

统一 bloom 模型 attention mask 的处理

### CODE REVIEW

File: paddlenlp/transformers/bloom/modeling.py [hunk 1]
```diff
@@ -181,33 +152,13 @@ def build_alibi_tensor(attention_mask: Tensor, num_heads: int, dtype) -> Tensor:
     # => the query_length dimension will then be broadcasted correctly
     # This is more or less identical to T5's relative position bias:
     # https://github.com/huggingface/transformers/blob/f681437203baa7671de3174b0fa583c349d9d5e1/src/transformers/models/t5/modeling_t5.py#L527
-    arange_tensor = ((attention_mask.cumsum(axis=-1) - 1) * attention_mask)[:, None, :]
+    arange_tensor = ((attention_mask.astype(paddle.float32).cumsum(axis=-1) - 1) * attention_mask)[:, None, :]
```
Comment (sijunhe) [inline paddlenlp/transformers/bloom/modeling.py:155]: 这里为啥要加一个float32的cast?

File: paddlenlp/transformers/bloom/modeling.py [hunk 2]
```diff
@@ -140,10 +114,7 @@ def _expand_mask(mask: Tensor, tgt_length: int) -> Tensor:
     batch_size, src_length = mask.shape[0], mask.shape[-1]
     tgt_length = tgt_length if tgt_length is not None else src_length
 
-    expanded_mask = ~(paddle.cast(mask[:, None, :], "bool"))
-    expanded_mask = paddle.cast(expanded_mask, dtype=paddle.float32)
-
-    return expanded_mask.expand([batch_size, tgt_length, src_length])
+    return mask.unsqueeze(axis=[1, 2]).expand([batch_size, 1, tgt_length, src_length])
```
Comment (sijunhe) [inline paddlenlp/transformers/bloom/modeling.py:117]: _expand_2d_mask

**Review by sijunhe — APPROVED** [review]
lgtm
